L1Trigger/L1TMuon/plugins/L1TMuonShowerProducer.cc

```diff
@@ -61,30 +61,45 @@ void L1TMuonShowerProducer::produce(edm::Event& iEvent, const edm::EventSetup& i
     Showers that arrive out-of-time are also under consideration, but are not
     going be to enabled at startup Run-3. So all showers should be in-time.
    */
-  bool isOneNominalInTime = false;
-  bool isTwoLooseInTime = false;
-  bool isOneTightInTime = false;
+  bool isOneNominalInTime{false};
+  bool isTwoLooseInTime{false};
+  bool isOneTightInTime{false};
+  bool isTwoLooseDifferentSectorsInTime{false};
 
+  bool foundOneLoose{false};
   for (size_t i = 0; i < emtfShowers->size(0); ++i) {
     auto shower = emtfShowers->at(0, i);
     if (shower.isValid()) {
       // nominal
-      if (shower.isOneNominalInTime())
+      if (shower.isOneNominalInTime()) {
         isOneNominalInTime = true;
+      }
       // two loose
-      if (shower.isTwoLooseInTime())
+      if (shower.isTwoLooseInTime()) {
         isTwoLooseInTime = true;
+      }
       // tight
-      if (shower.isOneTightInTime())
+      if (shower.isOneTightInTime()) {
         isOneTightInTime = true;
+      }
+      // two loos in different sectors
+      if (shower.isOneLooseInTime()) {
+        if (foundOneLoose) {
+          isTwoLooseDifferentSectorsInTime = true;
+        } else {
+          foundOneLoose = true;
+        }
```

Comment:
Is there any reason this shouldn't be combined with the 
```c++ 
if (shower.isOneLooseInTime())
```
 check above?

Reply:
The check in L78 is for TwoLoose (in the same sector). As for the data format it's true that those two conditions overlap, but I'd prefer to keep them separated because it's a bit clearer to me if that's fine for you.

Reply:
It's fine with me, it was just a question of organization.